todo_app/data/MongoDbApi.py

```diff
@@ -8,7 +8,7 @@
 
 class MongoDbApi:
 
-    MONGO_URL = 'cluster0.p7ckx.mongodb.net'
+    MONGO_URL = 'khalid-devops.mongo.cosmos.azure.com:10255'
```

Comment:
To avoid hardcoding this, it should be an environment variable. Otherwise what would you do if you want "test" and "production" versions of your app using different database servers?

Reply:
Moved to ENV variable as one connection string as part of project 12